pennylane/drawer/tape_mpl.py

```diff
@@ -53,6 +53,15 @@ def _add_multicontrolledx(drawer, layer, mapped_wires, op):
     drawer.CNOT(layer, mapped_wires, control_values=control_values)
 
 
+def _add_controlledqubitunitary(drawer, layer, mapped_wires, op):
```

Comment:
Instead of adding an additional case for `ControlledQubitUnitary`, we can check for the existence of `control_values` below where we add controls for arbitrary gates, see [master Line 287](https://github.com/PennyLaneAI/pennylane/blob/d539835b11457ae621cd4fab04f3d20184eeec5e/pennylane/drawer/tape_mpl.py#L287).

I'm working on adding in more general control operations, in which case *all* control operations will have `control_values`.  So we want to have general purpose logic for handling control values instead of gate-specific handling.

Reply:
I have updated the code to reflect these changes.
I am currently using:
```
control_values = "1" * len(control_wires)
```
for cases where control_values is None or not present. When all control operations have this attribute, this line can be deleted.